talent/job-descriptions/offers/qa-senior.md

```diff
@@ -0,0 +1,30 @@
+Senior QA Engineer
+------------------
+
+### [Location & Seniority](../location-seniority-section.md)
+
+**Compensation:** 49-53,000 Euros + 0.095% [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for senior engineers
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
```

Comment:
In this section, all engineering teams are described. Not sure if we should add QA as well, but I think for now we shouldn't as we still don't have the team composed.